examples/Python/python-django-webapp-sample/quickstartproject/settings.py

```diff
@@ -12,9 +12,15 @@
 
 import os
 from pathlib import Path
+import configparser
 from azure.appconfiguration.provider import load, AzureAppConfigurationKeyVaultOptions, SettingSelector
 from azure.identity import DefaultAzureCredential
 
+c_parser = configparser.ConfigParser()
+c_parser.read('static/config.ini')
+
+default = c_parser['DEFAULT']
```

Comment:
I will stick with the `config` variable name but add a comment explaining this is loading the default configuration from an ini file.